File: FE/lib/domain/entities/album_comment.dart
class AlbumComment {
  /// 댓글의 고유 식별자
  final int id;
  
  /// 댓글이 달린 앨범의의 ID
  final int memberId;
  
  /// 댓글 작성자의 닉네임
  final String nickname;
  
  /// 댓글 내용
  final String content;
  
  /// 댓글 작성 날짜 및 시간 (ISO 8601 형식)
  final String createdAt;

  final String userAvatar;

  /// 댓글 생성자
  const AlbumComment({
      required this.id,
      required this.memberId,
      required this.nickname,
      required this.content,
      required this.createdAt,
      required this.userAvatar,
  });
}

File: BE/ari/src/main/java/com/ccc/ari/community/domain/comment/entity/AlbumComment.java
package com.ccc.ari.community.domain.comment.entity;

import lombok.Getter;

import java.time.LocalDateTime;

/**
 * 앨범 댓글 도메인 엔티티
 */
@Getter
public class AlbumComment extends Comment {
    private final Integer albumId;

    public AlbumComment(Integer commentId, Integer albumId, Integer memberId, String content, LocalDateTime createdAt, Boolean deletedYn) {
        super(commentId, memberId, content, createdAt, deletedYn);
        this.albumId = albumId;
    }

    public static AlbumComment create(Integer albumId, Integer memberId, String content) {
        return new AlbumComment(null, albumId, memberId, content, LocalDateTime.now(), false);
    }
}
